fred2/missionsave.cpp

```diff
@@ -1937,8 +1937,13 @@ int CFred_mission_save::save_containers()
 					fout("\n+Strictly Typed Data");
 				}
 
+				SCP_vector<std::pair<SCP_string, SCP_string>> sorted_data(container.map_data.begin(), container.map_data.end());
+				std::stable_sort(sorted_data.begin(), sorted_data.end(),
```

Comment:
Although it doesn't matter, why `stable_sort`? Map keys must be unique.

Reply:
True.  This originated from when I was thinking of sorting lists as well as maps.  But you could say that specifying stable_sort here emphasizes the need to keep the written strings consistent.